test/integration/manifests/cnsconfig/ciliumconfigmap.yaml

```diff
@@ -27,5 +27,8 @@ data:
       "ProgramSNATIPTables" : true,
       "EnableCNIConflistGeneration": true,
       "CNIConflistFilepath": "/etc/cni/net.d/05-cilium.conflist",
-      "CNIConflistScenario": "cilium"
+      "CNIConflistScenario": "cilium",
+      "EnableAsyncPodDelete": true,
+      "WatcherPath": "/var/run/azure-vnet",
+      "Delete-Directory": "/deleteIDs"
```

Comment:
Should we keep the naming convention as `DeleteDirectory` as I don't see other attributes having `-` in them ?
Also, `DeletedIDsPath` or something like that would be better IMO as it is specifying the path for the deleted IDs.

Reply:
I'll keep the name the same for now as it represents the directory that will be placed inside the "WatcherPath" but thank's for catching the "-", it will be removed